datasets/batch_view/retention/intro.md

```diff
@@ -0,0 +1,84 @@
+The `retention` table provides client counts relevant to client retention at a
+1-day granularity. The project is tracked in [Bug 1381840][original_bug]
+
+### Contents
+
+The `retention` table contains a set of attribute columns used to specify a
+cohort of users and a set of metric columns to describe cohort activity. Each
+row contains a permutation of attributes, an approximation set of clients in a
```

Comment:
Should `approximation set` be `approximate count`?

Reply:
It should actually be `approximate set`. It's a set as opposed to a count since you can apply unions and intersections over the object.